library/src/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface ISelectContext {
+  selectedIds?: string[];
+  toggleSelect?: (id: string) => void;
+}
```

Comment:
I guess if I would leave out the `?`, then I would need to pass in these things already when declaring the context, correct? So

```
const SelectContext = React.createContext({ 
  selectedIds: [],
  toggleSelect: (id: string) => {},
 });
 ```

Reply:
correct